data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,455 @@ mission "Coalition: Outskirts Gauger Ready"
+		accept
+
+mission "Fangvin: Struggling Author - Human Media 1"
+	minor
+	name "Collect Human Media"
+	description "A group of Coalition citizens have asked you for some books from human space. Go to <destination> and buy as many books as possible."
+	source "Ki Patek Ka"
+	destination "Alexandria"
+	to offer
+		has "Wanderers: Translation Machine: done"
+		not "human cultural data"
+	on visit
+		dialog
+			`The station has a small museum gift shop that sells a copy of the entire archive on a data card. It only costs 40 credits, but because you have done a horrible job at managing your finances you do not have even that much cash on hand right now.`
+			`	Go earn some money, then return here.`
+	on offer
+		conversation "cultural data for coalition group"
+	on complete
+		set "human cultural data"
+		payment -40
+
+mission "Fangvin: Struggling Author - Data"
+	name "Cultural Data to Sol Kimek"
+	description "Bring back the datachip to <destination>, for the group of Coalition citizens who asked for it."
+	source "Alexandria"
+	destination "Ki Patek Ka"
+	on offer
+		conversation
+			`You had wondered how you would get a copy of the entire cultural archive, but it turns out to be easier than expected. The station has a small museum gift shop, which sells a copy of the archives on a data card. ("The perfect gift for the budding historian in your family! Hold all of human history in the palm of your hand. On sale now for only 39.99!") Given that a cheap data card costs a fraction of a credit, it's a bit of a rip-off, but you gladly buy one to take back to Eruk.`
+				accept
+
+event "coalition citizens discover fangvin"
+
+mission "Fangvin: Book Discovered"
+	name "Look for Anna Fangvin"
+	description "Get in contact with Anna Fangvin, an author in which the group of Coalition citizens have shown interest in."
+	source
+		government "Coalition"
+	destination "Zug"
+	to offer
+		has "event: coalition citizens discover fangvin"
+	on offer
+		conversation
+			`Upon landing in the spaceport, you get a message from the same group of Coalition citizens. You recall that you had provided them with a datachip containing the entire knowledge of humanity.`
+			``
+			`		Dear <first>,`
+			`	Interested in an author from the datachip, we are. 'Anna Fangvin', her name is. The story about your race's actions against the oppressors, captivate us it did. To converse with her, we wish.`
+			choice
+				`	(Disregard.)`
+					decline
+				`	(Look for Anna Fangvin.)`
+			`	Looking through your encyclopedia, you discover that Anna Fangvin is an author from <destination>. She is likely there. You decide to set a course for <destination>. You write a message to Anna explaining your intentions to meet with her for an interview, choosing not to mention the Coalition for now, as she would likely not believe it. As you are in Coalition space and therefore not connected to the human communications relay network, it will not send until you enter human space.`
```

Comment:
Whole section from here to end of this mission feels incredibly convoluted for just no real reason. Just have the mission take us to Zug because that's likely where she is, with an on complete to contact her hoping to set up appointment interview. She happens to be free (since she's struggling, nothing particularly eating up her time when you happen to arrive is highly plausible) and agrees to meet you in the spaceport. No need even to change start of next mission.